src/typing/typeloadFields.ml

```diff
@@ -1070,6 +1070,20 @@ let check_abstract (ctx,cctx,fctx) c cf fd t ret p =
 					error (cf.cf_name ^ ": Assignment overloading is not supported") p;
 				| (Meta.Op,[ETernary(_,_,_),_],_) :: _ ->
 					error (cf.cf_name ^ ": Ternary overloading is not supported") p;
+				| (Meta.Op,[EIs(_,_),_],_) :: _ ->
```

Comment:
Maybe use `@:is function ...` instead of `@:op(A is B)`? It's not really an _operator_ overloading and the second operand is fixed.

Reply:
I don't know :) Array access is technically also not an operator if we only consider operators EUnop and EBinop.

Reply:
btw `@:op(is)` could work too (similarly to `@:op([])`), but we want to make it a keyword someday, right?